src/Data/CurrentUser/CurrentUserDataItem.ts

```diff
@@ -124,29 +131,53 @@ export const CurrentUser = provideDataItem('CurrentUser', {
   },
 })
 
+async function getWhoAmIOnlyUser(): Promise<{
```

Comment:
* [ ] FYI The naming is not helpful. I had to read the callers and implementation to understand what it does. Given the trivial nature and a single caller we could inline the code to avoid naming issues.